embc-app/Controllers/VolunteersController.cs

```diff
@@ -47,10 +47,25 @@ public async Task<IActionResult> Get(string id)
             return Json(item);
         }
 
+        [HttpGet("bceid/{bceid}")]
+        public async Task<IActionResult> BceIdExists(string bceid)
+        {
+            if (string.IsNullOrWhiteSpace(bceid))
+            {
+                return BadRequest(Json(bceid));
+            }
+            var existing = await dataInterface.GetVolunteerByExternalIdAsync(bceid);
+            if (existing != null)
+            {
+                return BadRequest();
```

Comment:
This doesn't make sense, shouldn't it return boolean instead?

Reply:
This is how Alejandro wanted it.

If it returns a badrequest then the BCEID is safe to use if OK then a duplicate was found.

Reply:
@asanchezr , comment?

Reply:
Hey, what I suggested is to do something like this - https://softwareengineering.stackexchange.com/questions/347148/restful-service-for-checking-if-email-is-unique

I like the suggestion of doing the following:

GET /api/volunteers/bceid/xxxyyyzzz123

* Return 200 OK if bceid found
* Return 404 NOT FOUND if bceid not found

Reply:
@CodingVelocista 400!=404

Reply:
The above is generic enough and the front-end would translate the HTTP status to a validation message. So no need for a json response. Now the logic above is "inverted". What I would expect is:

```
if (existing != null) {
  return Ok();
}
return NotFound();
```

Reply:
I knew that!